### PR TITLE
[Avoiding Harm] Quizzes!

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?

There's some draft quizzes here. Ethics multiple choice questions are hard to make. They always feel a bit obvious. But Claude and I did our best.

### CODE REVIEW

**Comment by cansavvy** [comment]
Claude and I made these together. @carriewright11 do you want to review or do we merge and review later?

File: quizzes/02a-Avoiding_Harm-Overall_quiz.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+A) It can be unclear why the system made a certain decision
+b) The systems always behave as expected
+c) There are no risks associated with using them
+d) They cannot be trusted for any applications
+
+? According to the material, what should be considered at every stage of AI system development and use?
+
+a) Funding sources
+b) Legal protections
```
Comment (carriewright11) [inline quizzes/02a-Avoiding_Harm-Overall_quiz.md:24]:
```suggestion
b) Model usage
```

Comment (carriewright11) [reply quizzes/02a-Avoiding_Harm-Overall_quiz.md:24]: lol that might be true:P

File: quizzes/02a-Avoiding_Harm-Overall_quiz.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+a) It builds user trust in the systems
+b) It facilitates public monitoring and feedback
+c) It enables better understanding of AI-assisted decisions
+D) All of the above
+
+? What should be done to mitigate algorithmic bias?
+
+a) Nothing, AI systems are inherently unbiased
+B) Retrain models on more diverse and inclusive data
```
Comment (carriewright11) [inline quizzes/02a-Avoiding_Harm-Overall_quiz.md:45]:
```suggestion
B) Train models on more diverse and inclusive data
```

File: quizzes/02b-Avoiding_Harm-concepts_quiz.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+{quiz, id: societal_impact, attempts: 10}
+
+## Societal Impact Quiz
+
+? Which of the following is NOT considered a major ethical consideration discussed in the material?
+
+a) Security and Privacy issues
```
Comment (carriewright11) [inline quizzes/02b-Avoiding_Harm-concepts_quiz.md:8]:
```suggestion
a) Security and privacy issues
```

File: quizzes/02c-Avoiding_Harm-algorithms_quiz.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+b) Relying completely on AI tools without human review
+c) Assuming AI tools reflect current best practices
+D) Cross-checking responses from multiple AI tools
+
+? What does the term "hallucinate" refer to in the context of AI systems?
+
+a) AI systems having visions or dreams
+B) AI systems making up information based on artifacts of their algorithms
+c) AI systems malfunctioning due to hardware issues
+d) AI systems being trained on datasets containing hallucinogens
```
Comment (carriewright11) [inline quizzes/02c-Avoiding_Harm-algorithms_quiz.md:23]: lol I love this!

File: quizzes/02f-Avoiding_Harm-idare_and_ai.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+
+{quiz, id: idare_and_ai, attempts: 10}
+
+## IDARE and AI Quiz
+
+? What does the acronym IDARE stand for in the context of using AI responsibly?
```
Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:6]: ? What does the acronym IDARE stand for ?

Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:6]:
```suggestion
? What does the acronym IDARE stand for ??
```

File: quizzes/02f-Avoiding_Harm-idare_and_ai.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+? Why can bias occur in AI systems?
+A) Humans and human-created data are inherently unbiased
+b) AI systems are intentionally designed to discriminate
+c) The training data for AI systems is too diverse in representation
+d) Bias only occurs when using unsupervised learning
+
+? What example demonstrates an AI system perpetuating existing biases?
+a) An AI system beating a human at chess
+b) AI-written articles discussing IDARE principles
+c) Algorithms predicting patient ancestry from medical images
```
Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:22]:
```suggestion
c) Algorithms that help flag medical images for further review based on patient reported pain
```

File: quizzes/02f-Avoiding_Harm-idare_and_ai.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+D) Chatbots giving biased answers that lack diverse perspectives to users
```
Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:23]: D) An AI system that helps doctors suggest treatments for heart disease that doesn't work well as well for women

File: quizzes/02f-Avoiding_Harm-idare_and_ai.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+? What does the material recommend regarding using AI for employment decisions?
+a) AI should be fully relied upon to make unbiased choices
+b) Humans should have final say but AI can guide decisions
```
Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:27]:
```suggestion
b) Humans should have final say but AI can select a small number of resumes to evaluate without any potential bias issues
```

File: quizzes/02f-Avoiding_Harm-idare_and_ai.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+C) AI is not appropriate for use in employment decisions
```
Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:28]: C) AI is not appropriate for use in employment decisions with our current systems

Comment (carriewright11) [inline quizzes/02f-Avoiding_Harm-idare_and_ai.md:28]:
```suggestion
C) AI is not appropriate for use in employment decisions with our current systems
```

File: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+## Ethical Process Quiz
+
+? When should ethical considerations occur according to the proposed framework?
+
+a) Only after an AI system is already in use
+b) At the beginning of during development only
+c) Just before an AI system is deployed
+D) Throughout the entire process of development and use
+
+? What is one consideration when reflecting on AI use during the inception of an idea?
```
Comment (carriewright11) [inline quizzes/02g-Avoiding_Harm-ethical_process_quiz.md:13]:
```suggestion
? What is NOT a consideration when reflecting on AI use during the inception of an idea?
```

File: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+a) If larger training datasets are needed
+b) How transparency will be maintained
```
Comment (carriewright11) [inline quizzes/02g-Avoiding_Harm-ethical_process_quiz.md:15]:
```suggestion
b) Could you start smaller and build on your idea?
```

File: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+
+? When should ethical considerations occur according to the proposed framework?
+
+a) Only after an AI system is already in use
+b) At the beginning of during development only
+c) Just before an AI system is deployed
+D) Throughout the entire process of development and use
+
+? What is one consideration when reflecting on AI use during the inception of an idea?
+a) If larger training datasets are needed
```
Comment (carriewright11) [inline quizzes/02g-Avoiding_Harm-ethical_process_quiz.md:14]:
```suggestion
a) What training datasets are a reasonable size
```

File: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+a) Only after an AI system is already in use
+b) At the beginning of during development only
+c) Just before an AI system is deployed
+D) Throughout the entire process of development and use
+
+? What is NOT a consideration when reflecting on AI use during the inception of an idea?
+a) What training datasets are a reasonable size
+b) Could the project start smaller and be built up?
+c) Whether AI is actually appropriate for the use case
+d) If harmful content could be generated
```
Comment (carriewright11) [inline quizzes/02g-Avoiding_Harm-ethical_process_quiz.md:17]:
```suggestion
d) Should you start bigger?
```

File: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+? When should ethical considerations occur according to the proposed framework?
+
+a) Only after an AI system is already in use
+b) At the beginning of during development only
+c) Just before an AI system is deployed
+D) Throughout the entire process of development and use
+
+? What is NOT a consideration when reflecting on AI use during the inception of an idea?
+a) What training datasets are a reasonable size
+b) Could the project start smaller and be built up?
```
Comment (carriewright11) [inline quizzes/02g-Avoiding_Harm-ethical_process_quiz.md:15]:
```suggestion
b) What would the impact be if the AI system performed poorly
```

**Review by carriewright11 — APPROVED** [review]
Thank you so much, @cansavvy! This is great!!!

**Comment by carriewright11** [comment]
> Claude and I made these together. @carriewright11 do you want to review or do we merge and review later?

good job to you both haha - it looks good - I made some changes and added more, but it was pretty good!

**Pull request merged**